src/Symfony/Component/Notifier/Bridge/Lox24/README.md

```diff
@@ -0,0 +1,75 @@
+LOX24 SMS Notifier
+=========================
+
+Provides [LOX24 SMS Gateway](https://doc.lox24.eu/#tag/sms/operation/api_sms_post_collection) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+LOX24_DSN=lox24://TOKEN@default?from=FROM&type=SERVICE_CODE&voice_language=VOICE_LANGUAGE&is_text_delete=IS_TEXT_DELETE&callback_data=CALLBACK_DATA
+```
+
+where:
+
+- `TOKEN` (required) is LOX24 API v2 token.
+- `FROM` (required) is the sender of the message.
+- `TYPE` (optional) type of message: `sms` (by default) or `voice` (voice call).
+- `VOICE_LANGUAGE` (optional) if `type` is `voice`, then you can set the language of the voice message. Possible values: `de`, `en`, `es`, `fr`, `it` or `auto` (by default) per auto-detection.
+- `IS_TEXT_DELETE` (optional) delete SMS text from LOX24 database after sending SMS. Allowed values: `1` (true) or `0` (false). Default value: `0`.
+- `CALLBACK_DATA` (optional) additional data for the callback payload.
+
+See your account info at https://account.lox24.eu
+
+Send a Message
+---------------------------
+
+With a LOX24 Message, you can use the `LOX24Options` class to add
+message options.
+
+```php
+use Symfony\Component\Notifier\Message\SmsMessage;
+
+$sms = new SmsMessage('+1411111111', 'My message');
+
+$texter->send($sms);
+```
+
+Advanced Message options
+---------------------------
+
+With a LOX24 Message, you can use the `LOX24Options` class to add
+message options.
+
+```php
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Bridge\LOX24\LOX24Options;
+
+$sms = new SmsMessage('+1411111111', 'My message');
+
+$options = (new LOX24Options())
+    // set 'voice' per voice call (text-to-speech)
+    ->type('voice')
+    // set the language of the voice message.
+    // If not set or set 'auto', the automatic language detection by message text will be used
+    ->voiceLanguage('en')
+    // unix timestamp of SMS the delivery time. If 0 or not set, the message will be sent immediately
+    ->deliveryAt(1710871761)
+    // set True per delete the message from the LOX24 database after delivery
```

Comment:
```suggestion
    // set True to delete the message from the LOX24 database after delivery
```

This explanation/comment should be used in the options class as well, it is much more understandable

Reply:
done